<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>drmdecoder</class>
 <widget class="QWidget" name="drmdecoder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>drm decoder</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLCDNumber" name="show_int_offset">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&amp;quot;Coarse frequency offset as measured by the software&amp;quot;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="segmentStyle">
          <enum>QLCDNumber::Flat</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="show_small_offset">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Fine frequency offset as measured by the software (i.e. the offset to the nearest carrier).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="segmentStyle">
          <enum>QLCDNumber::Flat</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="countryLabel">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="channel_1">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>21</height>
          </size>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Visibility of the button suggests the availability of a DRM channel.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="timeLabel">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="channel_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="programTypeLabel">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="channel_3">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="aacDataLabel">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="channel_4">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>time sync</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="timeSyncLabel">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Time sync label, green means time synchronization is OK.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>FAC sync</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="facSyncLabel">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;FAC sync label, Green means that FAC's could be decoded (easy, often 4QAM).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>SDC sync</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="sdcSyncLabel">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;SDC sync label, green means SDC's could be decoded (usually 16 QAM).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>aac Sync</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="faadSyncLabel">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Faad sync label. Green means that the AAC decoder could decode audio.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QSpinBox" name="strengthSelector">
       <property name="maximum">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="f_cutSelector">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="spectrumIndicator">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="modeIndicator">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="datacoding">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="audioModelabel">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QComboBox" name="modeSelector">
       <item>
        <property name="text">
         <string>Pilots</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Channel</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Errors</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="fac_mer">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Estimate of the SNR of the DRM signal.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="sdc_mer">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="msc_mer">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="eqSelector">
       <property name="text">
        <string>linear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="selectedService">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="messageLabel">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QwtPlot" name="equalizerDisplay">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>131</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="scopeSlider">
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QwtPlot" name="iqPlotter">
       <property name="maximumSize">
        <size>
         <width>221</width>
         <height>141</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
